crates/lib/src/qpu/result_data.rs

```diff
@@ -0,0 +1,84 @@
+//! This modules provides types and functions for initializing and working with
+//! data returned from the QPU
+use enum_as_inner::EnumAsInner;
+use num::complex::Complex64;
+use quil_rs::instruction::MemoryReference;
+use std::collections::HashMap;
+
+use qcs_api_client_grpc::models::controller::{
+    readout_values as controller_readout_values, ReadoutValues as ControllerReadoutValues,
+};
+
+/// A row of readout values from the QPU.
```

Comment:
This would be a vector containing the values emitted for a particular memory reference (ie "ro[0]") across all shots. I added clarification to the docstring.